frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/health-card.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { DashboardCard } from '@console/internal/components/dashboard/dashboard-card/card';
+import { DashboardCardBody } from '@console/internal/components/dashboard/dashboard-card/card-body';
+import { DashboardCardHeader } from '@console/internal/components/dashboard/dashboard-card/card-header';
+import { DashboardCardTitle } from '@console/internal/components/dashboard/dashboard-card/card-title';
+import { HealthBody } from '@console/internal/components/dashboard/health-card/health-body';
+import { HealthItem } from '@console/internal/components/dashboard/health-card/health-item';
+import { HealthState } from '@console/internal/components/dashboard/health-card/states';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { STORAGE_HEALTH_QUERIES } from '../../../constants/storage';
+import { CEPH_HEALTHY, CEPH_DEGRADED, CEPH_ERROR, CEPH_UNKNOWN } from '../../../constants/index';
+
+const OCSHealthStatus = {
+  0: {
+    message: CEPH_HEALTHY,
+    state: HealthState.OK,
+  },
+  1: {
+    message: CEPH_DEGRADED,
+    state: HealthState.WARNING,
+  },
+  2: {
+    message: CEPH_ERROR,
+    state: HealthState.ERROR,
+  },
+  3: {
+    message: CEPH_UNKNOWN,
+    state: HealthState.ERROR,
+  },
+};
+
+const getCephHealthState = (ocsResponse: SystemHealth[]): CephHealth => {
+  if (!ocsResponse) {
+    return { state: HealthState.LOADING };
+  }
+  const value = _.get(ocsResponse, 'data.result[0].value[1]');
+  return OCSHealthStatus[value] || OCSHealthStatus[3];
+};
+
+const HealthCardMain: React.FC<DashboardItemProps> = ({
```

Comment:
@spadgett When I'm renaming it too `HealthCard_` then the build is failing due to lint errors. I can see the underscore being used in the public folder. So do we need to enable it for the package folder too?


@rawagner ^^
```
/go/src/github.com/openshift/console/frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/details-card.tsx
  45:7  error  Unexpected dangling '_' in 'DetailsCard_'  no-underscore-dangle
 /go/src/github.com/openshift/console/frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/health-card.tsx
  45:7  error  Unexpected dangling '_' in 'HealthCard_'  no-underscore-dangle
```

Reply:
@christianvogt @vojtechszocs ^ Do we want to keep this lint rule? I suspect it's to avoid using `_` to indicate "private" class methods, but it's not how we're using it here.

Reply:
+1 for turning the rule off
@christianvogt  @vojtechszocs can you state your preference ?

Reply:
> @christianvogt @vojtechszocs ^ Do we want to keep this lint rule? I suspect it's to avoid using `_` to indicate "private" class methods, but it's not how we're using it here.

> @christianvogt @vojtechszocs can you state your preference ?

I agree with @christianvogt, code under `packages` should follow single default component export per file convention. With that, the trailing underscore isn't needed and therefore not an issue.

Please see my comment https://github.com/openshift/console/pull/1873#issuecomment-507736616

> If some package isn't ready to follow that convention, it can add e.g. `packages/foo-pkg/.eslintrc` file that extends `packages/.eslintrc.js` and suppresses no-underscore-dangle until its code is updated.